presto-docs/src/main/sphinx/security/cli.rst

```diff
@@ -0,0 +1,124 @@
+===========================
+CLI Kerberos Authentication
+===========================
+
+The Presto :doc:`/installation/cli` can connect to a :doc:`Presto coordinator
+</security/server>` that has Kerberos authentication enabled.
+
+Environment Configuration
+-------------------------
+
+.. |subject_node| replace:: client
+.. |subject_ref| replace:: cli
+
+.. include:: kerberos-services.fragment
+.. include:: kerberos-configuration.fragment
+
+Kerberos Principals and Keytab Files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Each user who connects to the Presto coordinator needs a Kerberos principal.
+You will need to create these users in Kerberos using `kadmin
+<http://web.mit.edu/kerberos/krb5-latest/doc/admin/admin_commands/kadmin_local.html>`_.
+
+Additionally, each user needs a `keytab file
+<http://web.mit.edu/kerberos/krb5-devel/doc/basic/keytab_def.html>`_. The
+keytab file can be created using :command:`kadmin` after you create the
+principal.
+
+.. code-block:: none
+
+  kadmin
+  > addprinc -randkey someuser@EXAMPLE.COM
+  > ktadd -k /home/someuser/someuser.keytab someuser@EXAMPLE.COM
+
+.. include:: ktadd-note.fragment
+
+.. include:: jce-policy.fragment
+
+Java Keystore File for TLS
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Access to the Presto coordinator should be through https when using Kerberos
```

Comment:
Changed here and in server.rst: https://github.com/prestodb/presto/pull/5217/files#diff-23116ae070edd519249cb243ed857298R65